plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/embeddings/callables/SpecialFunctions.kt

```diff
@@ -8,10 +8,29 @@ package org.jetbrains.kotlin.formver.embeddings.callables
 import org.jetbrains.kotlin.formver.embeddings.LegacyUnspecifiedFunctionTypeEmbedding
 import org.jetbrains.kotlin.formver.embeddings.VariableEmbedding
 import org.jetbrains.kotlin.formver.names.AnonymousName
+import org.jetbrains.kotlin.formver.names.GetterFunctionName
+import org.jetbrains.kotlin.formver.names.GetterFunctionSubjectName
 import org.jetbrains.kotlin.formver.names.SpecialName
-import org.jetbrains.kotlin.formver.viper.ast.BuiltinFunction
-import org.jetbrains.kotlin.formver.viper.ast.Declaration
-import org.jetbrains.kotlin.formver.viper.ast.Type
+import org.jetbrains.kotlin.formver.viper.MangledName
+import org.jetbrains.kotlin.formver.viper.ast.*
+import org.jetbrains.kotlin.formver.viper.ast.Function
+
+class FieldAccessFunction(
```

Comment:
Could you add a comment that documents what this function generates?  I think it may be non-obvious when reading back later.